base/container_scripts/db_snapshot.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+declare FILENAME="$1"
+
+set -e
+
+mkdir -p /opt/oci_env/db_backup/
+
+echo "dumping database to /var/lib/pulp"
+pg_dump -U pulp -F c -b -f "/var/lib/pulp/pulp_db.backup" 
```

Comment:
Why these two stages? Can't we just pipe the result of pg_dump into gzip directly?

Reply:
because it's creating a tar of everything in `/var/lib/pulp/`, not just the database dump. The database is dumped into /var/lib/pulp/ and then the entire directory (including all the artifacts) is archived.